client/src/components/Form/CheckboxGroup.js

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import FormLabel from "@material-ui/core/FormLabel";
+import FormControl from "@material-ui/core/FormControl";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import Checkbox from "@material-ui/core/Checkbox";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    paddingLeft: theme.spacing(10),
+  },
+  formControl: {
+    margin: theme.spacing(3),
+  },
+  formLabel: {
+    margin: "25px 0",
+    textAlign: "start",
+  },
+  formHelperText: {
+    color: "red",
+    margin: "10px 0",
+  },
+  checkboxGroup: {
+    flexDirection: "row",
+    justifyContent: "space-around",
+  },
+  checkboxLabel: {
+    width: 185,
+  },
+}));
+
+export default function CheckboxesGroup({ checkboxData, setCheckboxData }) {
+  const classes = useStyles();
+
+  const handleChange = (event) => {
+    setCheckboxData({
+      ...checkboxData,
+      [event.target.name]: event.target.checked,
+    });
+  };
+
+  const {

```

Comment:
Will these stay hardcoded or will we be able to populate the list depending on what they offer, even if it's outside of the list?

Reply:
They are just hardcoded for now to use as a boilerplate